contrib/gitea-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,228 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local prometheus = grafana.prometheus;
+
+{
+
+  grafanaDashboards+:: {
+
+    local giteaSelector = 'job="$job", instance="$instance"',
+    local giteaStatsPanel = grafana.statPanel.new(
+                              'Gitea stats',
```

Comment:
Too much indentation?

Reply:
Agree, too much, it had been generated by jsonnetfmt so I made some adjustments to make sure`make fmt` stick 2-spcae indendation as much as possible.